sentry-anyhow/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+//! Adds support for capturing Sentry errors from `anyhow::Error`.
+//!
+//! # Example
+//!
+//! ```no_run
+//! # use sentry_core as sentry;
+//! # fn function_that_might_fail() -> anyhow::Result<()> { Ok(()) }
+//! use sentry_anyhow::capture_anyhow;
+//! # fn test() -> anyhow::Result<()> {
+//! let result = match function_that_might_fail() {
+//!     Ok(result) => result,
+//!     Err(err) => {
+//!         capture_anyhow(&err);
+//!         return Err(err);
+//!     }
+//! };
+//! # Ok(()) }
+//! ```
+
+#![deny(missing_docs)]
+#![deny(unsafe_code)]
+
+use std::error::Error;
+use std::fmt;
+
+use sentry_core::internals::Uuid;
+use sentry_core::Hub;
+
+/// Captures an error chain.
+pub fn capture_anyhow(e: &anyhow::Error) -> Uuid {
+    Hub::with_active(|hub| hub.capture_anyhow(e))
+}
+
+/// Hub extension methods for working with anyhow.
```

Comment:
_nit_: Put `anyhow` in code quotes.

```suggestion
/// Hub extension methods for working with `anyhow`.
```